solr/views.py

```diff
@@ -70,6 +70,8 @@ def cleanup_solr_request(payload):
                 fields = filter(lambda x: x not in disallowed, fields)
             if len(fields) == 0:
                 fields.append('id')
+            if '*' in fields:
+                fields = current_app.config.get('SOLR_SERVICE_ALLOWED_FIELDS')
             payload['fl'][0] = ','.join(fields)
```

Comment:
wow, i didn't know '*' was valid. But there are two more edge cases to take care of. Could you pls change also two other lines?

```fields = [x.strip() for x in fields[0].split(',')]```

and 

```payload['fl'] = ','.join(fileds)```

just to be sure